index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <!--
+
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
+    <script  src="js/index.js"></script>
+
+    <header>
+      <h1 id="title"><title>Survey Form</title>Survey Form</h1>
+      <p id="description" > "What do you thing about our services"</p>
+    </header>
+    <main>
+      <form id="survey-form">
+l       <label id="name-label" for="name" >Please enter your name:</label>
+        <input id="name" type="text" required placeholder="Alejandro Maggioni">
+
```

Comment:
Could you check line 32 does not contain unwanted characters.

Reply:
- [ ] This comment is a little confusing. It says "does not contain unwanted characters"  instead of "it contains the unwanted character". Kindly update the comment to avoid any confusion. 💯 
 - [ ] Also, it would be nice if you could also specify the consequences of leaving the unwanted characters in the codebase to the user 😄 
 - [**OPTIONAL**] How about making the comments more human-friendly? I think it would be nice if you  could add some emojis or gifs whenever possible in your comments to make it human-friendly 😃